src/power/PowerComponentWin.h

```diff
@@ -9,6 +10,14 @@ class PowerComponentWin : public PowerComponent
   PowerComponentWin() : PowerComponent(0) { }
   virtual void doDisableScreensaver();
   virtual void doEnableScreensaver();
+
+  virtual bool canPowerOff() override { return WinUtils::getPowerManagementPrivileges(); }
```

Comment:
Yeah who knows with windows API. if we only need to call it once, let's just call it once.